### ISSUE TITLE
@include path uniqueness

### ISSUE THREAD

**Issue by mm-gmbd** [electricimp/Builder#36]
The following paths are all considered unique:

```
@include once "/a/path/to/a/file.nut"
@include once "a/path/to/a/file.nut"
@include once "./a/path/to/a/file.nut"
```
I believe builder automatically references all "@include"'s to the root of the project, so these should all be considered the same -- as it stands today, 3 copies of the file will be included in the resultant build.

**Comment by ppetrosh** [comment]
I believe it's going to be addressed as a part of #35. Please let me know if you think otherwise.

**Comment by betzrhodes** [comment]
Fixed in release [v2.6.0](https://github.com/electricimp/Builder/releases/tag/v2.6.0)
